### PR TITLE
9968 Update open-vm-tools to 10.3.5, 64-bit-only, GTK+3

### PR DESCRIPTION
**Pull request by Mno-hime**
Release notes: https://github.com/vmware/open-vm-tools/blob/stable-10.3.5/ReleaseNotes.md

### CODE REVIEW

**Comment by Mno-hime** [comment]
Tested with VMware ESXi 6.7U1: General information from `vmtoolsd` service, screen resolution change, managed restart/shutdown.

image

~~I added workaroung for `vmtoolsd` to find plugins as it now looks for them (incorrectly) in `usr/lib/$(MACH64)/open-vm-tools/plugins/amd64/`.~~

File: components/sysutils/open-vm-tools/open-vm-tools.p5m [hunk 1]
```diff
@@ -65,6 +65,7 @@ file path=usr/lib/$(MACH64)/libhgfs.so.0.0.0
 link path=usr/lib/$(MACH64)/libvmtools.so target=libvmtools.so.0.0.0
 link path=usr/lib/$(MACH64)/libvmtools.so.0 target=libvmtools.so.0.0.0
 file path=usr/lib/$(MACH64)/libvmtools.so.0.0.0
+link path=usr/lib/$(MACH64)/open-vm-tools/plugins/$(MACH64) target=.
```
Comment (pyhalov) [inline components/sysutils/open-vm-tools/open-vm-tools.p5m]: Should we do it? This looks ugly.
Better patch open-vm-tools-stable-10.3.0/services/vmtoolsd/pluginMgr.c:
It has 
```
#if defined(sun) && defined(__x86_64__)
   const char *subdir = "/amd64";
#else
   const char *subdir = "";
#endif
```
Just set it to ""

Comment (Mno-hime) [reply components/sysutils/open-vm-tools/open-vm-tools.p5m]: Thanks for finding the culprit. I removed the uglyness and re-tested.

Comment (pyhalov) [reply components/sysutils/open-vm-tools/open-vm-tools.p5m]: Sorry, I don't see the patch in PR.

Comment (Mno-hime) [reply components/sysutils/open-vm-tools/open-vm-tools.p5m]: Thanks for catching it, forgot to add it.

**Pull request merged**